tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -192,6 +192,11 @@ internal void SubscribeToChanges(params string[] productNames)
             }
         }
 
+        internal ApplyDetails[] UpdateFromRcmForTest(Dictionary<string, List<RemoteConfiguration>> configsByProduct)
```

Comment:
I don't really like this but it's a fast and effective way to test this code... The alternatives for unit testing would be declaring UpdateFromRcm internal or use reflection in the tests.

Reply:
FWIW, this is fine IMO 🙂 If you want to make sure people don't call it by accident, you could add `[Obsolete]` or `[PublicApi]`